src/templates/ar/2019/base.html

```diff
@@ -97,10 +146,45 @@
         var menuIcon = document.querySelector('.menu-icon');
         var menuIconInitialSrc = document.querySelector('.menu-icon').src;
         var menuIconSrcClose = document.querySelector('.menu-icon').dataset.srcClose;
+        var menuBtn = document.querySelector('.menu-btn');
+        var menuNav = document.querySelector('#Menu');
+        var menu = document.querySelector('#Menu');
+
+        function triggerMenu() {
+          menuBtn.click();
```

Comment:
I think this is a case for naming the click event handler and invoking that function directly, rather than going through the event loop.

Reply:
Sorry I'm not sure what this means? Can you explain to someone who's JavaScript is less than stellar?

Reply:
Sure, for example we'd bind the click event listener like this:

```js
function toggleNavMenu() {
  // implement the existing handler code here...
}

menuBtn.addEventListener('click', toggleNavMenu);
```

Then in places where you're calling `menuBtn.click()` or `triggerMenu()` instead you'd do `toggleNavMenu()`.

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/666/commits/1d5172131674bfb0a4e5f21ed6431d4a09e56eef